src/sentry/static/sentry/app/components/narrowLayout.jsx

```diff
@@ -1,15 +1,41 @@
 import jQuery from 'jquery';
 import React from 'react';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import {Client} from 'app/api';
+import styled from 'react-emotion';
+
+const Logout = styled.span`
+  float: right;
+  padding-top: 3px;
+  font-size: 16px;
```

Comment:
I believe you can use `font-size: inherit` here.

Reply:
Hm... I tried `font-size: inherit` but it went back to the 20px header font size. I think 16px which is the size of the modal text looks better.

Reply:
yeah i was hoping it would inherit global size but sounds like that's a no. it's fine to keep it.